### ISSUE TITLE
is the library working today

### ISSUE THREAD

**Issue by ammarbhatkar** [ryanheise/just_waveform#33]
iam trying to use with just audio library of yours 
1) the combination of both will work?
2) iam trying to run your example code but giving error :T
arget of URI doesn't exist: 'package:just_waveform/just_waveform.dart'.
Try creating the file referenced by the URI, or try using a URI for a file that does

**Comment by ryanheise** [comment]
Just to be clear, are you running the example directly out of this repository, i.e. the one inside the example directory and after running flutter pub get? Because what you're running doesn't sound exactly like my example.

If you are experimenting with your own project, it just be because you didn't set up your project's pub dependencies correctly

**Comment by ammarbhatkar** [comment]
Yes it is working.. but can we use this library " just waveform". with the
library just audio so that we can track the progress of audio  using the
wave form created .. for eg in WhatsApp like when we send an audio message
and the user plays it the waveform is different for every audio and as it
is played the progress is shown

**Comment by ryanheise** [comment]
I can't offer that kind of advice here. You can review the API documentation of this plugin and see if it meets your requirements. If you find the API is enough for what you're trying to implement, then you can answer your question that way. If the API is not enough for what you're trying to implement, you can think about what specific changes you would like me to make to the API and submit a feature request.
